curriculum/challenges/english/09-information-security/information-security-projects/stock-price-checker.english.md

```diff
@@ -23,6 +23,11 @@ Start this project on Repl.it using <a href="https://repl.it/github/freeCodeCamp
 
 ```yml
 tests:
+  - text: I can provide my own project, not the example URL.
+    testString: |
+      getUserInput => {
+        assert(!/.*\/stock-price-checker\.freecodecamp\.rocks/.test(getUserInput('url')));
```

Comment:
```suggestion
        assert(!/.*\/stock-price-checker-proxy\.freecodecamp\.rocks/.test(getUserInput('url')));
```

Reply:
Wait I thought the `proxy` url was for the API, not the example project.

Reply:
@nhcarrigan Yes, it is. That is why I made this suggestion...to immediately remove it 🤦‍♂️